http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -354,7 +356,7 @@ class GKEInterpreter[F[_]](
       )
 
       // Create KSA
-      ksaName: ServiceAccountName <- F.fromOption(
+      ksaName <- F.fromOption(
```

Comment:
`:` inside for comprehension will introduce an additional call...we should avoid doing that...ascribing type anywhere else is encouraged :)